### PR TITLE
Update build files

### PR DESCRIPTION
**Pull request by dev0tion**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # you can use {branch} name in version format too
 # version: 1.0.{build}-{branch}
 
-version: 0.2.0.{build}
+version: 0.1.0.{build}
```
Comment (bokobza) [inline appveyor.yml]: we can even bump this to 0.3.0.

File: appveyor.yml [hunk 2]
```diff
@@ -120,20 +119,20 @@ test_script:
...
-#- provider: GitHub
-#  tag: $(APPVEYOR_REPO_TAG_NAME)
-#  auth_token:
-#    secure: faXallYrtShJywqNK+adalNNl50Wjns7dFsCDDx5/eNBegshCLjfshUeMPgIX9Ld
-#  draft: false
-#  prerelease: true
-#  force_update: true
-#  on:
-#    branch: master                 # release from master branch only
+# deploy:
```
Comment (bokobza) [inline appveyor.yml]: Please uncomment this and I'll make the necessary change for it to be deployed properly.

**Comment by bokobza** [comment]
btw you also need the electron-builder file and to update the main.ts file.

File: appveyor.yml (hunk 1 above)
Comment (dev0tion) [reply appveyor.yml]: Done

**Pull request merged**